apertium-tur-uzb.tur-uzb.dix

```diff
@@ -301,7 +302,7 @@
     <!-- SECTION: Determinatives -->
     
 <e><p><l>bir<s n="det"/><s n="ind"/></l><r>bir<s n="det"/><s n="ind"/></r></p></e>
-<e><p><l>herhangi<s n="det"/><s n="ind"/></l><r>har<b/>qanday<s n="det"/><s n="ind"/></r></p></e>
+<e><p><l>herhangi<s n="det"/><s n="ind"/></l><r>har<b/>qanday<s n="det"/><s n="ind"/></r></p></e>  <!-- Will be moved to MWE later-->
```

Comment:
why do you want to remove this? it could be prn itg

Reply:
Well, remove what?, "herhangi"?. I did not remove it, just added a comment at the end, since it's a multi word element.